### PR TITLE
IHub-182879 Move the filling of some fields to the rule

### PR DESCRIPTION
**Pull request by P4radise**

### CODE REVIEW

**Review by glebowadim — APPROVED** [review]

File: import_kml.py [hunk 1]
```diff
@@ -98,66 +97,23 @@ def parse(self, kml_filename):
...
         for placemark in parse.find_all('Placemark'):
             name = placemark.find('name')
             if name is None:
                 raise Exception(f'Failed parse KML. Exception [Field name not found]')
 
-            name = name.text
-
             description = placemark.find('description')
             if description is None:
                 raise Exception(f'Failed parse KML. Exception [Field description not found]')
```
Comment (asmoiseenko) [inline import_kml.py:107]: what is the problem to run with an empty description?

**Review by asmoiseenko — CHANGES_REQUESTED** [review]

File: import_kml.py (hunk 1 above)
Comment (P4radise) [reply import_kml.py:107]: The problem here is not that the description may be empty, but that this field may not be found. But this field will have some data in any case, if it is found in kml.

Comment (P4radise) [reply import_kml.py:107]: The problem here is not that the description may be empty, but that this field may not be found. But this field will have some data in any case, if it is found in kml.

File: import_kml.py [hunk 2]
```diff
@@ -98,66 +97,24 @@ def parse(self, kml_filename):
...
-            fire_size = re.search(r'\d+', fire_size_acres.group())
-            if fire_size is None:
-                raise Exception(f'Failed parse KML. Exception [Number in fire_size field is not found]')
-
-            fire_size = fire_size.group()
-
-            coordinates = placemark.find('Point').find('coordinates')
-            if coordinates is None:
-                raise Exception(f'Failed parse KML. Exception [Field coordinates not found]')
+                self.integration_log.add(LogLevel.WARNING, f'Failed parse KML. Exception [Field description not found for {name}]')
```
Comment (asmoiseenko) [inline import_kml.py:108]: correct error message to something more relevant: "Description field is not found for XXX, it will be skipped"

**Review by asmoiseenko — CHANGES_REQUESTED** [review]

File: import_kml.py (hunk 2 above)
Comment (P4radise) [reply import_kml.py]: Done

**Review by asmoiseenko — APPROVED** [review]

**Pull request merged**